gnat2goto/driver/tree_walk.adb

```diff
@@ -335,6 +335,42 @@ package body Tree_Walk is
 
    procedure Remove_Entity_Substitution (E : Entity_Id);
 
+   function Create_Dummy_Irep return Irep;
+
+   procedure Report_Unhandled_Node_Empty (N : Node_Id;
+                                          Fun_Name : String;
+                                          Message : String);
+   function Report_Unhandled_Node_Irep (N : Node_Id;
+                                        Fun_Name : String;
+                                        Message : String) return Irep;
+   function Report_Unhandled_Node_Kind (N : Node_Id;
+                                        Fun_Name : String;
+                                        Message : String) return Irep_Kind;
+
+   procedure Report_Unhandled_Node_Empty (N : Node_Id;
+                                          Fun_Name : String;
+                                          Message : String) is
+   begin
+      Put_Line (Standard_Error, "----------At: " & Fun_Name & "----------");
+      Put_Line (Standard_Error, "----------" & Message & "----------");
+      pp (Union_Id (N));
```

Comment:
One thing to note here is that `pp` will actually print to stdout rather than stderr. Hopefully there's a way to get the same thing for stderr, but I haven't found it yet.